tekton/src/tasks/fvt/fvt-manage-pytest.yml.j2

```diff
@@ -108,7 +108,17 @@ spec:
         value: "$(params.fvt_enable_debug)"
 
       - name: ARTIFACTORY_TOKEN
-        value: $(params.artifactory_token)
+        valueFrom:
```

Comment:
@unnati-solanki-git as you are updating the token here as well, we need also to remove it from the task parameter here https://github.com/ibm-mas/cli/pull/792/files#diff-810eda5557acae5fc32b41243809e5f54e69c2ca5ed9056a26cc80f9552a5f97R64 and make sure the task defintions where this pytest taskref is invoked is no longer sending the parameter to it.

Reply:
okay let me remove this for now, will refactor it later.